feature-examples/gradle.properties

```diff
@@ -20,7 +20,7 @@
 # limitations under the License.
 #
 version=1.0.0-build.0
-gemfireVersion=10.1.+
```

Comment:
Does this bump the gemfireVersion for all examples? Do we want to increment this with this PR given that 10.2 is not yet available?

Reply:
I changed it back, because ci will always run on 10.0.